coala_quickstart/green_mode/green_mode.py

```diff
@@ -0,0 +1,537 @@
+import fnmatch
+import itertools
+import os
+import yaml
+from copy import deepcopy
+from pathlib import Path
+
+from coala_quickstart.generation.Utilities import (
+    split_by_language, peek, contained_in)
+from coala_quickstart.generation.SettingsClass import (
+    SettingTypes)
+from coala_quickstart.green_mode.QuickstartBear import (
+    QuickstartBear)
+from coalib.bears.GlobalBear import GlobalBear
+from coalib.processes.Processing import (
+    get_file_dict, yield_ignore_ranges)
+from coalib.settings.Section import Section
+
+
+settings_key = 'green_mode_infinite_value_settings'
+
+
+def get_contents(project_data):
```

Comment:
should have yaml in the name, and belongs in Utilities somewhere outside green mode.